dist/.magento.env.yaml

```diff
@@ -262,6 +262,20 @@
 #                elasticsearch_index_prefix: magento                                                                  #
 #                elasticsearch_server_timeout: '15'                                                                   #
 #######################################################################################################################
+# ELASTIC_SUITE_CONFIGURATION - use this variable to retain customized ElasticSearch service settings                 #
+# between deployments.                                                                                                #
+# Magento Version: 2.1.4 and later                                                                                    #
+# Default value: not set                                                                                              #
+# Stages: deploy                                                                                                      #
+# Example:                                                                                                            #
+#          stage:                                                                                                     #
+#            deploy:                                                                                                  #
+#              ELASTIC_SUITE_CONFIGURATION:                                                                           #
```

Comment:
just noticed. Why elasticsuite was split with underscore? It should be one word ELASTICSUITE_CONFIGURATION

Reply:
Fixed